pkg/local_object_storage/pilorama/interface.go

```diff
@@ -0,0 +1,35 @@
+package pilorama
+
+import cidSDK "github.com/nspcc-dev/neofs-sdk-go/container/id"
+
+// Forest represents CRDT tree.
+type Forest interface {
```

Comment:
All the methods have `Tree` name prefix, can we do without it?

Reply:
We can, but it would be confusing on `StorageEngine` IMO. We already have `metabase.MoveIt` and it is a completely different thing.
Another option is to have `(*StorageEngine).Pilorama()` which returns a new type implementing methods without prefix.